### PR TITLE
Feature: integrate external

### LINKED ISSUE #445: Integrate external_id changes
## Description

Integrate external_id changes.

Update the documentation about plugin-release.

### PR DESCRIPTION
**Pull request by aboubakar14**
resolve #445

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -31,14 +31,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-  * Work with [githubator-plugin](https://github.com/ditrit/githubator-plugin/tree/0.3.0) version 0.3.0.
+  * Work with [plugin-core](https://github.com/ditrit/leto-modelizer-plugin-core/tree/0.23.0) version 0.23.0.
+  * Work with [terrator-plugin](https://github.com/ditrit/terrator-plugin/tree/0.8.0) version 0.8.0.
+  * Work with [githubator-plugin](https://github.com/ditrit/githubator-plugin/tree/0.3.1) version 0.3.1.
 * Improve performance on git add.
 * Replace OIDC authentication by Leto-Modelizer-Api, see [this issue](https://github.com/ditrit/leto-modelizer/issues/425).
 * Separating git-related functions into a dedicated composable.
 * Merge add/create diagram buttons into a drop down button, see [this issue](https://github.com/ditrit/leto-modelizer/issues/471).
 * Improve Authentication by re-doing login process if the token is expired, see [this issue](https://github.com/ditrit/leto-modelizer/issues/478).
+* Handle external id
```
Comment (justinehell) [inline changelog.md:42]: missing `.`

File: README.md [hunk 2]
```diff
@@ -100,8 +100,8 @@ Plugin to manage github action files.
     <tr>
       <td>next version</td>
       <td>= 0.22.0</td>
-      <td>= 0.7.1</td>
-      <td>= 0.3.0</td>
+      <td>= 0.8.0</td>
+      <td>= 0.3.1</td>
```
Comment (justinehell) [inline README.md:104]: update plugins' version but not plugin-core ? (0.22.0 -> 0.23.0)

File: .github/workflows/CI.yml [hunk 3]
```diff
@@ -159,8 +159,8 @@ jobs:
         run: |
           npm install -g npm@8.19.3 &&  \
           npm ci && \
-          npm run plugin:install -- repository-name="githubator-plugin" repository-url="https://github.com/ditrit/githubator-plugin.git#0.3.0" && \
-          npm run plugin:install -- repository-name="terrator-plugin" repository-url="https://github.com/ditrit/terrator-plugin.git#0.7.1" && \
+          npm run plugin:install -- repository-name="githubator-plugin" repository-url="https://github.com/ditrit/githubator-plugin.git#0.3.1" && \
+          npm run plugin:install -- repository-name="terrator-plugin" repository-url="https://github.com/ditrit/terrator-plugin.git#0.8.0" && \
```
Comment (justinehell) [inline .github/workflows/CI.yml:163]: maybe change commit name because its also updating the plugins version

**Review by justinehell — CHANGES_REQUESTED** [review]

File: changelog.md (hunk 1 above)
Comment (Zorin95670) [reply changelog.md:42]: done

File: README.md (hunk 2 above)
Comment (Zorin95670) [reply README.md:104]: done

File: .github/workflows/CI.yml (hunk 3 above)
Comment (Zorin95670) [reply .github/workflows/CI.yml:163]: done

**Review by Zorin95670 — DISMISSED** [review]

**Review by justinehell — DISMISSED** [review]

**Review by Zorin95670 — APPROVED** [review]

**Review by Zorin95670 — APPROVED** [review]

**Pull request merged**